check_replication_queue.py

```diff
@@ -0,0 +1,32 @@
+from client import get_client
+import sys
+import waiting
+import click
+
+
+SNUBA_ADMIN_HOST = "127.0.0.1"
+CLICKHOUSE_CLIENT_PORT = "9000"
+
+def _is_replication_queue_empty(client):
+    return len(client.execute("SELECT * FROM system.replication_queue LIMIT 1;")) == 0
```

Comment:
if we don't need the values from the table we could just use `select count(*)` instead

Reply:
I think `count(*)` would return a row with the value 0, so even if the `replication_queue` is empty, `len(client.execute("SELECT * FROM system.replication_queue LIMIT 1;"))` would be 1.